### DISCUSSION TITLE
Image enhancement nvenc h264

### DISCUSSION THREAD

**Discussion by shershunov** [obsproject/obs-studio#7743] [Development Questions] [answered]
Hi, I'm streaming on Twitch, since I don't have a second computer, I have to stream through the nvenc codec. The quality in dynamic games comes out slightly blurry. I saw that you added a double pass in the last update. I have a fairly powerful video card, it would be possible to make a setup for people with a similar situation thanks to which the quality will improve by reducing performance.

I would be very grateful, and I'm not the only one)

**Comment by Fenrirthviti** [comment, marked as answer]
This sounds more likely to be a support request, and we are not currently accepting support requests on GitHub Discussions. Make sure you read any [pinned posts](https://github.com/obsproject/obs-studio/discussions/5206) on discussion boards before opening a new discussion. Please use our [forums](https://obsproject.com/forum/) or [Discord](https://obsproject.com/discord/) for further assistance.

Thank you!
